cmd/openshift-install/internal_integration_test.go

```diff
@@ -570,3 +639,27 @@ func lookForCpioFiles(r io.Reader) ([]string, error) {
 
 	return files, nil
 }
+
+// In the CI environment, a dynamically generated pull secret is used to access the release image,
+// which is necessary for running integration tests. Since the value of this pull secret cannot be
+// known or hardcoded in the test data, comparisons involving the pull secret are skipped when OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE is set.
+func skipPullSecretCmp(ts *testscript.TestScript, aFilePath, eFilePath string) bool {
```

Comment:
by performing the substitution in the agent pull secret file (when available, just in the ztp test case), also this one should not very likely required